### DISCUSSION TITLE
supported postgres versions 13-16?

### DISCUSSION THREAD

**Discussion by christiangierschner** [datasharingframework/dsf#222] [General]
Hi,
we use our own docker-compose.yml setup and i've seen that nowhere are supported plattforms mentioned.
Do you support postgres version 13-16? Are there Limitations?

**Comment by schwzr** [comment]
Hi!
In theory, PostgreSQL 13 and 14 should also be compatible. However, we are only testing from PostgreSQL 15 and also reserve the right to use features that are only available from PostgreSQL 15 onwards.
From our point of view, PostgreSQL 16 should be compatible without any issues.
Newer major DSF versions may then also require newer PostgreSQL versions. These (DSF 2.x) will then be announced accordingly and provided with migration instructions.
